src/ui/osx/TogglDesktop/test2/AppDelegate.m

```diff
@@ -92,6 +93,7 @@ @implementation AppDelegate
 void *ctx;
 BOOL manualMode = NO;
 BOOL onTop = NO;
+BOOL unsupportedOS = NO;
```

Comment:
Pls remove this unnecessary state. Ideally, we shouldn't introduce more `State`, as more as `State`, more maintenance cost.